mongoapi/dbtestapp/Dockerfile

```
@@ -4,15 +4,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM golang:1.18.0-stretch AS test
+FROM golang:1.21.3-bookworm AS test
```

To avoid a dependebot PR to update this to the latest version :-)
```suggestion
FROM golang:1.21.6-bookworm AS test
```

---

Done